app/models/user.rb

```diff
@@ -2,7 +2,7 @@ class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :validatable
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
- [ ] Good job adding the confirmation, the link to confirm get correctly sent on console, but when clicked it should display the page, instead, I am getting this error 
![image](https://user-images.githubusercontent.com/62705947/224008238-d83906e3-3a18-466e-983d-44dfaf6d1f72.png)

Reply:
Thank you, I have fixed the confirmation link issue.